plugins/main/bridge/bridge_test.go

```diff
@@ -2211,3 +2248,50 @@ var _ = Describe("bridge Operations", func() {
 		}
 	})
 })
+
+func assertMacSpoofCheckRulesExist() {
+	assertMacSpoofCheckRules(
+		func(actual interface{}, expectedLen int) {
+			ExpectWithOffset(3, actual).To(HaveLen(expectedLen))
+		})
+}
+
+func assertMacSpoofCheckRulesMissing() {
+	assertMacSpoofCheckRules(
+		func(actual interface{}, _ int) {
+			ExpectWithOffset(3, actual).To(BeEmpty())
+		})
+}
+
+func assertMacSpoofCheckRules(assert func(actual interface{}, expectedLen int)) {
```

Comment:
I don't know what is the policy this project has for testing dependencies of a unit test.
I believe it is enough to verify `Setup` and `Teardown` were invoked. `spoofcheck_test.go` already testing the internal rules.

Reply:
These are the integration tests, which check the plugin from the API up to the actual effect on the kernel/system.
The unit tests against `spoofcheck` do not require kernel support for nftables, nor they check the outcome validity. They just check that the intention of the code correspond to the actual outcome.

The tests here, are inclusive of the nftables validation and ability to apply then on a system.
We are not suppose to mock/stub here anything, just provide input values and expect the system to be populated with the expected results.